Lib/inspect.py

```diff
@@ -2334,7 +2346,11 @@ def p(name_node, default_node, default=empty):
                 default = ast.literal_eval(default_node)
             except ValueError:
                 raise ValueError("{!r} builtin has invalid signature".format(obj)) from None
-        parameters.append(Parameter(name, kind, default=default, annotation=empty))
+        try:
+            annotation = parse_annotation(name_node)
```

Comment:
I'd prefer if this directly passed in `name_node.annotation`, instead of branching inside `parse_annotation`

Reply:
Ok, I did this.  (some asserts were moved)